### ISSUE TITLE
Sending OPENAI KEY to fleet

### ISSUE THREAD

**Issue by maralski** [fleet-ai/context#22]
I was looking to add direct support for Anthropic but then I stumbled on this code.

https://github.com/fleet-ai/context/blob/4b978da81831095d3aead82d82c991cd2b6355b3/utils/ai.py#L36

Are you able to explain what you do with the LLM keys sent to your endpoint?

**Comment by tavisrudd** [comment]
Yikes. It's also in `context.query` so anyone following along with the examples in the readme will unwittingly send their API key to you. Why do you need this? If it's absolutely required, this should be highlighted in bold red at the top of the readme.

**Comment by big-nico** [comment]
Hey Peter and Tavis - really appreciate your help pointing this out. This was clearly a major blunder on our end. We previously used the CLI as an internal tool our team and had LLM generation happening on the API. 

When we spun it out publicly, we moved the generation logic to the CLI but accidentally left in the parameter for the API key in the retrieve function. 

Good thing is that we did not track the API keys nor did we even use them in the retrieval process. I have updated the CLI & our backend API to remove the API key reference. 

On behalf of the entire fleet team, we really apologize for this.
